### DISCUSSION TITLE
Give morale bonus when eating food/drink meant to be served cold

### DISCUSSION THREAD

**Discussion by scarf005** [cataclysmbn/Cataclysm-BN#2487] [General]
for example, fizzy drinks and iced creams.

 we could implement it similar to how heating food works

**Comment by chaosvolt** [comment]
To enable this, we'll likely want the restoration of food actually tracking temperature status. My proposal has generally been that the best approach would be re-implement the temperature feature, but rework "frozen" status into "very cold" instead, so that it doesn't block eating the item, doesn't affect transfer of liquids, and doesn't inflict mushy penalties if it thaws out.

The other big benefit of this is players will be able to actually tell at a glance if cold environment or a freezer is actually preserving their food as intended, which would've been very useful for debugging previous issues with freezers, and still remains useful for preventing questions about whether their fridge is doing anything or not since then they'll be able to see if it's actually doing anything.

If anyone could link me to which PR or commit changed all this it might be easy enough to fix up, but I've not had any luck finding it if I recall.

**Reply by scarf005** [reply]
i think it'd be easiest to go the #2572 route. #2250's acting as a blocker, tho.
